Assets/MixedRealityToolkit-Preview/CameraCapture/CameraCaptureARFoundation.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if USE_ARFOUNDATION
+
+using System;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+using UnityEngine;
+using UnityEngine.XR.ARExtensions;
+using UnityEngine.XR.ARFoundation;
+
+namespace Microsoft.MixedReality.Toolkit.CameraCapture
```

Comment:
I wasn't sure about this either! I figured if we did that, then whenever it gets removed from Preview it would break any user code that references it. That seems a little unneccessary, but maybe that would be the point?

@davidkline-ms Your thoughts?

Reply:
I added 'Preview' to the namespaces!